includes/Share.php

```diff
@@ -91,7 +91,7 @@ public static function onSidebarBeforeOutput( Skin $skin, &$sidebar ) {
 
 					if ( $wgShareReddit ) {
 						$sidebar['share-header'][] = [
-							'html' => '<a href="https://www.reddit.com/submit?url=' . urlencode( $currenturl ).'"><img src="'.$wgExtensionAssetsPath.'/Share/resources/images/reddit.png" alt="'.$skin->msg( 'share-reddit' ).'" width="90" height="30"></a>',
+							'html' => '<a href="https://www.reddit.com/submit?url=' . urlencode( $currenturl ).'"><img src="'.$wgExtensionAssetsPath.'/Share/resources/images/reddit.png" alt="'.wfMessage( 'share-reddit' )->plain().'" width="90" height="30"></a>',
```

Comment:
Why not `$skin->msg( 'share-reddit' )->escaped()`?

Reply:
When I first considered this, since I saw ->escaped() allowed for templates to be called and transformed, I thought that wouldn't be necessary and preferred ->plain() instead since it doesn't allow for templates to be invoked but now that I think about, there could be legitimate use cases for invoking a template so I guess I'll change it to ->escaped().